study-builder/studies/osteo/study.conf

```diff
@@ -2400,7 +2400,8 @@
         ! (user.studies["CMI-OSTEO"].forms["PARENTAL_CONSENT"].hasInstance()
         && user.studies["CMI-OSTEO"].forms["RELEASE_MINOR"].hasInstance()
         && user.studies["CMI-OSTEO"].forms["PARENTAL_CONSENT"].isStatus("COMPLETE")
-        && user.studies["CMI-OSTEO"].forms["RELEASE_MINOR"].isStatus("COMPLETE"))
+        && user.studies["CMI-OSTEO"].forms["RELEASE_MINOR"].isStatus("COMPLETE")
+        && !user.studies["CMI-OSTEO"].forms["RELEASE_SELF"].hasInstance())
```

Comment:
needed to check if minor is aged-up and if so cancel pdf generation with minor template

Reply:
I think I get what we're trying to say here, maybe it's clearer to write expression as below?
```
! ( has_parental()...
&& is_complete())
|| has_release_self()
```